homeassistant/util/aiohttp.py

```diff
@@ -37,6 +37,9 @@ def enable_chunking(self) -> None:
     async def write_headers(self, *args: Any, **kwargs: Any) -> None:
         """Write headers."""
 
+    async def write(self, *args: Any, **kwargs: Any) -> None:
+        """Write payload."""
```

Comment:
This extension was needed to avoid the following error

```
    async def write(self, data: Union[bytes, bytearray, memoryview]) -> None:
        assert isinstance(
            data, (bytes, bytearray, memoryview)
        ), "data argument must be byte-ish (%r)" % type(data)
    
        if self._eof_sent:
            raise RuntimeError("Cannot call write() after write_eof()")
        if self._payload_writer is None:
            raise RuntimeError("Cannot call write() before prepare()")
    
>       await self._payload_writer.write(data)
E       AttributeError: 'MockPayloadWriter' object has no attribute 'write'

/home/vscode/.local/ha-venv/lib/python3.13/site-packages/aiohttp/web_response.py:550: AttributeError
```